s3torchconnector/tst/unit/test_s3_client_config.py

```diff
@@ -0,0 +1,42 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  // SPDX-License-Identifier: BSD
+from hypothesis import given
+from hypothesis.strategies import integers, floats
+
+from s3torchconnector import S3ClientConfig
+
+
+def test_default():
+    config = S3ClientConfig()
+    assert config is not None
```

Comment:
Not blocking: The is not None check isn't needed. It's definitely going to be an instance of `S3ClientConfig`. At the very most I'd do an isinstance check